book/src/filters.md

```diff
@@ -104,7 +104,7 @@ Output:
 ### default
 [#default]: #default
 
-```jinja
+```text
```

Comment:
Is it possible to make this ```` ```jinja,ignore ````? One does not see it on the stable book branch, but on latest the jinja code is highlighted: <https://askama.readthedocs.io/en/latest/template_syntax.html>. It would be sad if the highlighting was disabled for half of the examples.

Hm, I don't know if highlight.js works a comma separated list, but I guess it's part of commonmark standard?

Reply:
Fair enough.